cpp/CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ if (UPPERCASE_CMAKE_BUILD_TYPE MATCHES "DEBUG")
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g")
 
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fsanitize=address -fno-omit-frame-pointer ")
+#    # TODO SANDEEPA REVERT!
+#    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fno-omit-frame-pointer ")
```

Comment:
is this intentded?

Reply:
I kept it there because there were some errors generated from the original line when testing locally. I'll remove it.